uint8 START=0
uint8 STOP=1
uint8 recorder_action

---

string response_message